src/routes/docs/comparison-with-other-tools.mdx

```diff
@@ -55,25 +57,7 @@ Some other advantages Mirage has:
   server.get("/api/movies", () => ["Interstellar", "Inception"])
   ```
 
-  MSW requires some additional setup. Their docs have you generate a file that needs to be in the public project of your dev setup, and that you also need to ensure is stripped from production builds. You can then setup a worker in development, but because the service worker is asynchronous, you need to wait for it to finish before rendering your app.
-
-  Here's an example from the docs:
-
-  ```js
-  import { worker } from "./mocks"
-
-  async function bootstrap() {
-    await worker.start()
-
-    ReactDOM.render(<App />, document.getElementById("root"))
-  }
-
-  bootstrap()
-  ```
-
-  The service worker, while enabling usage of devtools, does add a bit more complexity to the overall integration. For example you also need to ensure your service worker [updates on source code reloads](https://redd.gitbook.io/msw/getting-started#chrome).
-
-- Because there is no service worker barrier, it's trivial to interact with Mirage's server and DB during development. People have even made dev-time tools to tweak their server at runtime – for example, a floating React component with a slider bound to Mirage's timing delays.
+  MSW requires one additional step - you need to generate a file that in the public directory of your project, which you might also want to strip from production builds.
```

Comment:
Could we mention that it's a `Service Worker file`? Just to be clear what the sentence is about.